Projects/UOContent/Gumps/RunebookGump.cs

```diff
@@ -7,504 +7,448 @@
+                                if (!HasSpell(from, 209))
                                 {
                                     from.SendLocalizedMessage(500015); // You do not have that spell!
+                                    _book.Openers.Remove(from);
+                                    break;
                                 }
 
-                                Book.Openers.Remove(from);
+                                SendTargetCoordinates(from, e.Location, e.Map);
 
-                                break;
+                                _book.OnTravel();
+                                new SacredJourneySpell(from, e).Cast();
                             }
-                        case 5: // Sacred Journey
-                            {
-                                if (Core.AOS)
-                                {
-                                    if (HasSpell(from, 209))
-                                    {
-                                        int xLong = 0, yLat = 0;
-                                        int xMins = 0, yMins = 0;
-                                        bool xEast = false, ySouth = false;
-
-                                        if (Sextant.Format(
-                                            e.Location,
-                                            e.Map,
-                                            ref xLong,
-                                            ref yLat,
-                                            ref xMins,
-                                            ref yMins,
-                                            ref xEast,
-                                            ref ySouth
-                                        ))
-                                        {
-                                            var location =
-                                                $"{yLat}° {yMins}'{(ySouth ? "S" : "N")}, {xLong}' {xMins}'{(xEast ? "E" : "W")}";
-                                            from.SendMessage(location);
-                                        }
-
-                                        Book.OnTravel();
-                                        new SacredJourneySpell(from, e).Cast();
-                                    }
-                                    else
-                                    {
-                                        from.SendLocalizedMessage(500015); // You do not have that spell!
-                                    }
-                                }
 
-                                Book.Openers.Remove(from);
+                            _book.Openers.Remove(from);
 
-                                break;
-                            }
-                    }
-                }
-                else
-                {
-                    Book.Openers.Remove(from);
+                            break;
+                        }
                 }
             }
+            else
+            {
+                _book.Openers.Remove(from);
```

Comment:
Actually, I think openers is always removed. Is there a case where it isn't?

Reply:
it's not removed in the cases where you receive a new `RunebookGump` in return for your response, but I see how I can clean it up